subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationResult.java

```diff
@@ -256,8 +336,8 @@ public void visitInputFileProperties(final InputFilePropertyVisitor visitor) {
             .orElse(null);
     }
 
-    @SuppressWarnings("deprecation")
     @Override
+    @SuppressWarnings({"deprecation", "RedundantSuppression"})
```

Comment:
I think this line should be removed now.

Reply:
done